nupic/algorithms/anomaly.py

```diff
@@ -84,22 +93,15 @@ def __init__(self, slidingWindowSize=None, mode=MODE_PURE, binaryAnomalyThreshol
               models probability of receiving this value and anomalyScore
           - "weighted" - "pure" anomaly weighted by "likelihood"
```

Comment:
It would be better to fix the docstring to refer to `Anomaly.MODE_WEIGHTED`, etc., instead of the raw strings "weighted", "likelihood", etc.